sky/check.py

```diff
@@ -80,14 +83,51 @@ def check_one_cloud(cloud_tuple: Tuple[str, clouds.Cloud]) -> None:
     global_user_state.set_enabled_clouds(enabled_clouds)
 
 
+def _no_public_cloud_enabled_locally() -> bool:
+    locally_enabled_clouds = global_user_state.get_locally_enabled_clouds()
```

Comment:
How about renaming it to `cached_enabled_clouds` and also `get_cached_enabled_clouds`?